app/controllers/passengers_controller.rb

```diff
@@ -15,56 +17,41 @@ def index
   end
 
   def show
-    @passenger = Passenger.find(params[:id])
   end
 
   def edit
-    @passenger = Passenger.find(params[:id])
   end
 
   def create
     @passenger = Passenger.new(passenger_params)
-
-    respond_to do |format|
-      if @passenger.save
-        format.html { redirect_to @passenger, notice: 'Passenger was successfully created.' }
-        format.json { render :show, status: :created, location: @passenger }
-      else
-        format.html { render :new }
-        format.json { render json: @passenger.errors, status: :unprocessable_entity }
-      end
+    if @passenger.save
+      redirect_to @passenger, notice: 'Passenger was successfully created.'
+    else
+      render :new
     end
   end
 
   def update
-    respond_to do |format|
-      @passenger = Passenger.find(params[:id])
-      if @passenger.update(passenger_params)
-        format.html { redirect_to @passenger, notice: 'Passenger was successfully updated.' }
-        format.json { render :show, status: :ok, location: @passenger }
-      else
-        format.html { render :edit }
-        format.json { render json: @passenger.errors, status: :unprocessable_entity }
-      end
+    if @passenger.update(passenger_params)
+      redirect_to @passenger, notice: 'Passenger was successfully updated.'
+    else
```

Comment:
Rubocop would prefer inline here:

```ruby
else render :edit
```

I definitely do recommend incorporating it sooner rather than later.